farmanager/_run.demo.bat

```diff
@@ -0,0 +1,3 @@
+call _presetvars.bat
```

Comment:
This script may be transfered to  `test.ps1` to have a simple test in AppVeyor build.